### DISCUSSION TITLE
Add custom attributes and slots to Brands

### DISCUSSION THREAD

**Discussion by glennjacobs** [lunarphp/lunar#1018] [Roadmap]
Developers often want to use brands for different purposes. Introducing custom attributes and slots gives developers the further flexibility they may need.

**Comment by alecritson** [comment]
Sounds like a good idea, currently we have the `name` column. Would we still look to keep this or convert it to a `name` attribute?

**Reply by glennjacobs** [reply]
We'd keep the `name` field, like we did with Customers.

**Comment by glennjacobs** [comment]
Pretty sure this is done now.
